packages/mask/src/plugins/Tips/storage/index.ts

```diff
@@ -22,6 +25,9 @@ export function getStorage() {
     return storage.storage
 }
 
+export const TIPS_GUIDE_TOTAL = 1
+export const TIPS_GUIDE_INIT = 0
```

Comment:
Please use an enum stead.

Reply:
I think this should not use enum, the TOTAL is step's sum.